### DISCUSSION TITLE
clarification on module documentation.

### DISCUSSION THREAD

**Discussion by ssagerian** [AllskyTeam/allsky-modules#85] [General]
The current documentation on modules describes the clear sky, star count and meteor detection modules. I dont see them in the list of modules to install when running the installer. Have they been renamed or remove or something else? thanks
Steve

**Comment by Alex-developer** [comment]
They are included with the core Allsky installation now. I'll update the documentation

**Reply by ssagerian** [reply]
great, how do load them ? I dont see them in the module editor or as modules waiting to be enabled?  I also ran the module installer, lots of good modules there but no "core"?

**Reply by Alex-developer** [reply]
If you are running version  v2023.05.01_04  or later they should be there. In all versions prior to  v2023.05.01_04  you will need to enable experimental modules in the module manager settings

**Comment by Alex-developer** [comment]
Closing, if you are still having an issue please feel free to reopen
